docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     env_file:
       - .env
     container_name: 'cdapython-notebook'
-    image: jupyter/minimal-notebook:latest
+    image: jupyter/minimal-notebook:2023-05-29
```

Comment:
Why is this changed to a specific version?

Reply:
This change was made because  the latest image uses python 3.11. Currently this branch dose not support 3.11 but a future branch will.